acceptance/tests/config-entries/config_entries_namespaces_test.go

```diff
@@ -90,6 +90,10 @@ func TestControllerNamespaces(t *testing.T) {
 
 				"global.acls.manageSystemACLs": strconv.FormatBool(c.secure),
 				"global.tls.enabled":           strconv.FormatBool(c.secure),
+
+				"terminatingGateways.enabled":              "true",
+				"terminatingGateways.gateways[0].name":     "terminating-gateway",
```

Comment:
🧠  Curious how this works if `terminatingGateways` isn't already set to a list with >= 1 items (or is it?)

Reply:
do you mean this specific string or do you mean how the code being tested behaves?

Reply:
walked through on a call. setting this here isn't actually _updating_ something but instead uses magic to _create_ something behind the scenes